kythe/cxx/indexer/cxx/IndexerPPCallbacks.h

```diff
@@ -82,7 +82,7 @@ class IndexerPPCallbacks : public clang::PPCallbacks {
   /// \sa clang::PragmaHandler::HandlePragma
   void HandleKytheMetadataPragma(clang::Preprocessor& Preprocessor,
                                  clang::PragmaIntroducerKind Introducer,
-                                 clang::Token& FirstToken);
```

Comment:
As much as I prefer `lower_with_unders` this should be `FirstToken` to be consistent with the rest of the file.